### DISCUSSION TITLE
package development with pluto

### DISCUSSION THREAD

**Discussion by drdozer** [fonsp/Pluto.jl#1350] [Questions about using Pluto]
Hi - I am just starting out with package development in Julia, and to be honest, with Julia. The model I attempted and failed at was to start to develop a new package, and then to test out ideas in a notepad session. However, I wasn't using Pluto so all my cells got stale, and I couldn't figure out how to get Revise to ping updates to my module source code into the notepad environment.

Is there a recipe for how to achieve this with Pluto? So the workflow I'm hoping for would look like:

1. load up my module source in vscode
2. open a new (or existing) pluto notebook
3. write notebook cells that exercise/text my module API, and be able to associate these back to the module source
4. fix bugs, add API to my module, and see the notebook re-evaluate

**Comment by fonsp** [comment]
Also see https://github.com/JuliaPluto/PlutoTest.jl and #508
